actions/check_version/index.js

```diff
@@ -0,0 +1,19 @@
+const core = require("@actions/core");
+const github = require("@actions/github");
+const { exec } = require("child_process");
+try {
+  exec("pod ipc spec kintone-ios-sdk.podspec", function(err, stdout, stderr) {
```

Comment:
@Pham-Gia-Huong 
make "kintone-ios-sdk.podspec" dynamic instead of fixed (get from action input)